src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,120 @@
+package lambda.part1.exercise;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.arrayContaining;
+
+class Exercise1 {
+
+    @Test
+    void sortPersonsByAgeUsingArraysSortLocalComparator() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, new PersonComparator<>());
+
+        assertThat(persons, is(arrayContaining(
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        )));
+    }
+
+    private static class PersonComparator<T extends Person> implements Comparator<T> {
```

Comment:
That's not a local class. You need to move it inside the `sortPersonsByAgeUsingArraysSortLocalComparator()`.

Reply:
done